radio/src/Makefile

```diff
@@ -850,6 +856,8 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
   ifeq ($(RTCLOCK), YES)
     CPPDEFS += -DRTCLOCK
     CPPSRC += rtc.cpp targets/sky9x/rtc_driver.cpp
+  else ifeq ($(PCB), SKY9X)
+    CPPSRC += targets/sky9x/rtc_driver.cpp
```

Comment:
Yes it seems `rtc_driver.cpp` and `coproc_driver.cpp` actually contain a bunch of duplicated or nearly similar code, not sure why `rtc_driver.cpp` even exists for sky9x. If you're up for cleaning this up, removing dupes and putting the rest in `coproc_driver.cpp`it would be welcome. 
The `#if defined(RTCLOCK)` would then become `#if defined (COPROCESSOR)` and it would all be more clear and clean. Might require a few fixes in different places.
If you decide to fix that it might be wise to do it as a separate step in another branch after this one is merged.